### DISCUSSION TITLE
How do I transfer Arch from one user of win10 to another user?

### DISCUSSION THREAD

**Discussion by GitYun** [yuk7/ArchWSL#232] [Q&A] [answered]
I installed WSL2 Arch in win10 user "user1" before, and now I have another user "user2", but I still want to use WSL Arch of the "user1" in "user2", what should I do?

**Comment by yuk7** [comment, marked as answer]
Hi @GitYun 
I think there are two ways.
### You can back up your file system and restore it to user2.
1. Please run this command in user1 to backup:
```
Arch.exe backup
```
2. Please copy backup.tar.gz or backup.ext4.vhdx and restore it in user2:
```
Arch.exe install backup.tar.gz
```
```
Arch.exe install backup.ext4.vhdx
```


### If you want to use one system in common (a little dangerous)
1. Give user2 full access to the ArchWSL installation directory.
(Note: If you use AppX pkg, it's `C:\Users\{Your-User-Name}\AppData\Local\Packages\yuk7.archwsl_35zwpb4sx6e50\LocalState`)
2. backup registry file in user1
```
Arch.exe backup --reg
```
3. restore registry file in user2 (Please double-click reg file)

**Comment by GitYun** [comment]
@yuk7 
Thanks! I use the second way, It works!!!
